src/Microsoft.ML/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- https://learn.microsoft.com/en-us/dotnet/fundamentals/package-validation/diagnostic-ids -->
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>CP0004</DiagnosticId>
+    <Target>Microsoft.ML, Version=1.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51</Target>
+    <Left>lib/netstandard2.0/Microsoft.ML.dll</Left>
+    <Right>right</Right>
+    <IsBaselineSuppression>true</IsBaselineSuppression>
+  </Suppression>
```

Comment:
Consider adding a comment. Is this because you dropped netstandard2.0 from the current packages? There's now also the option with the new APICompat to ignore baseline TFMs.

Reply:
This is here because I fixed `Microsoft.ML` to no longer build an empty assembly.  We wanted to do that before but couldn't as there was no good way to do this in a normal CSProj while still packing symbols, but the NoTargets SDK lets us do that.  I can add a comment.